templates/partials/definition.html

```diff
@@ -0,0 +1,33 @@
+{% from "components/collapsible/_macro.njk" import onsCollapsible %}
+
+{% set definition_id = definition.title | lower %}
+
+{% call onsCollapsible({
+  "id": "definition-" ~ definition_id,
+  "title": definition.title,
+  "classes": "u-mb-s",
+  "headingAttributes": {
+    "data-ga": "click",
+    "data-ga-category": "Definition",
+    "data-ga-action": "Open panel",
+    "data-ga-label": definition_id,
+    "data-qa": "definition-" ~ definition_id ~ "-title"
+  },
+  "contentAttributes": {
+    "data-qa": "definition-" ~ definition_id ~ "-content"
+  },
+  "button": {
+    "close": _("Hide this"),
+    "contextSuffix": "content",
+    "attributes": {
+      "data-ga": "click",
+      "data-ga-category": "Definition",
+      "data-ga-action": "Close panel",
+      "data-ga-label": definition_id,
+      "data-qa": "definition-"  ~ definition_id ~ "-button"
+    }
+  }
+})%}
+  {% set contents = item.definition.contents %}
+  {% include 'partials/contents.html' %}
+{% endcall %}
```

Comment:
I think this partial should be called for `question-definitions.html` to reduce the duplication as they are pretty much identical. Only thing you have to do is pass the id in as a parameter.

Reply:
Its not just the id, data-ga-category, data-qa and the loop index would also need adding

Reply:
The loop index is the same as the id. i.e `"question-definition-" ~ loop.index` is also the ID which could be passed in. 
If the ga category really needs to be "question definition" we could pass that in, it is 1 extra attribute only to make a total of 2.